timemachine/cpp/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ string(APPEND CMAKE_CUDA_FLAGS
   " -Wextra"
   " -Wno-unused-parameter"
   " -Wreorder"
+  " -Wfloat-conversion"
```

Comment:
What about `-Wconversion`?

Reply:
(I'll definitely take `-Wfloat-conversion` as well)

Reply:
It throws a lot of warnings at the moment mostly about going from `size_t -> int`. Should work towards enabling `-Wconversion`.